requirements.in

```diff
@@ -9,5 +9,4 @@ pydantic
 psycopg2-binary
 sqlalchemy
 raven[flask]
-f8a_worker @ git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@066c2f6#egg=f8a_worker
-f8a_utils @ git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git@5a5ce60#egg=f8a_utils
+f8a_worker @ git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@cf7b8a7#egg=f8a_worker
```

Comment:
Since you are using f8a_utils types, it should be a direct dependency described over here.